docs/nydus-fscache.md

```diff
@@ -85,20 +92,49 @@ For more information on how to configure containerd to use nydus snapshotter ple
  --log-to-stdout
 ```
 
-4. Run container with [ctr-remote](../contrib/ctr-remote)
+5. Configure containerd to use `nydus-snapshotter` by editing
+   `/etc/containerd/config.toml` like below:
+
+``` toml
+version = 2
+
+[plugins]
+  [plugins."io.containerd.grpc.v1.cri"]
+    [plugins."io.containerd.grpc.v1.cri".cni]
+      bin_dir = "/usr/lib/cni"
+      conf_dir = "/etc/cni/net.d"
+  [plugins."io.containerd.internal.v1.opt"]
+    path = "/var/lib/containerd/opt"
+
+[proxy_plugins]
+  [proxy_plugins.nydus]
+    type = "snapshot"
+    address = "/run/containerd/containerd-nydus-grpc.sock"
 
-```shell
+[plugins."io.containerd.grpc.v1.cri".containerd]
+   snapshotter = "nydus"
+   disable_snapshot_annotations = false
+```
+
+For more information on how to configure containerd to use nydus snapshotter please refer to [here](./containerd-env-setup.md).
+
+6. Restart containerd with
+   `service containerd restart`
+
+7. Run container with [ctr-remote](../contrib/ctr-remote)
+
+``` shell
 # pull nydus image
-ctr-remote images rpull hsiangkao/ubuntu:20.04-rafs-v6-docker
+contrib/ctr-remote/bin/ctr-remote images rpull docker.io/hsiangkao/ubuntu:20.04-rafs-v6-docker
 
 # run nydus image
-ctr-remote run --rm -t --snapshotter=nydus hsiangkao/ubuntu:20.04-rafs-v6-docker ubuntu /bin/bash
+ctr run --rm -t --snapshotter=nydus docker.io/hsiangkao/ubuntu:20.04-rafs-v6-docker ubuntu /bin/bash
```

Comment:
We can also run with `ctr-remote`, because `ctr-remote` has already been compiled, so just use it.

Reply:
ctr doesn't installed by default? I think using the default tool is always perferred, no?

Reply:
Well, if we assume that `ctr` is already installed, we can use it. `ctr-remote` is actually the same as ctr except a new `rpull` subcommand.

Reply:
Yeah, I agree, is there some cases that `ctr` didn't install together with `containerd`?